components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -66,7 +66,7 @@ @interface MDCBottomNavigationBar () <MDCInkTouchControllerDelegate>
 @property(nonatomic, strong) UIView *itemsLayoutView;
 @property(nonatomic, strong) NSMutableArray *inkControllers;
 @property(nonatomic) BOOL shouldPretendToBeATabBar;
-@property(nonatomic, strong) UILayoutGuide *barItemsLayoutGuide;
+@property(nonatomic, strong) UILayoutGuide *barItemsLayoutGuide NS_AVAILABLE_IOS(9_0);
```

Comment:
Did this introduce warnings on iOS 8?

Reply:
UILayoutGuide was introduced in iOS 9, so to followup on @romoore 's comment.
Do we need to add guards around this property?

Reply:
It did when I enabled this flag.

Reply:
Where is the use unguarded?  As far as I know, those methods are guarded by availability checks.  Is this warning flag generating errant reports?

Reply:
Ping @jverkoey

Reply:
<img width="630" alt="screen shot 2019-03-08 at 9 55 04 am" src="https://user-images.githubusercontent.com/45670/54035875-4fb30200-4188-11e9-89ba-95fe454795ab.png">

Reply:
Looks like you took away the AVAILABLE macro?

Reply:
Ah, no, sorry. I guess I missed declaring it as such. 

My earlier comments meant to refer to the conditionals below.

Reply:
Wait, what?! That layout guide should not have been merged...

Reply:
Haah let's chat offline.

Reply:
Ah, I guess the warning is just being silly since the type won't be accessed in the run time.